project/CodeCoverageSettings.scala

```diff
@@ -16,9 +16,11 @@ object CodeCoverageSettings {
     "testOnly.*",
     "testOnlyDoNotUseInAppConf.*"
   )
+  private val excludedFiles: Seq[String] = Seq(".*group_created.template")
```

Comment:
I assume we'll remove this later? Although how come it's just the template file 🤔

Reply:
yeah. templates with little content fail because it tries to include the apply() function.